src/components/Sidebar/index.jsx

```diff
@@ -61,25 +69,35 @@ export default class Sidebar extends Component {
 
     // Add Swift directly as there is no `lang=swift` in bugzilla whiteboards
     return (
-      <List disablePadding>
-        {['Swift', ...Object.keys(BUGZILLA_LANGUAGES)].map(language => (
-          <ListItem
-            className={classNames({
-              [classes.active]: language.toLowerCase() === activeLanguage,
-            })}
-            button
-            onClick={this.props.onLanguageClick}
-            id={language}
-            key={language}
-            component={Link}
-            to={`/languages/${language.toLowerCase()}`}>
-            <ListItemIcon>{icons[language] || <WebIcon />}</ListItemIcon>
-            <ListItemText disableTypography className={classes.text}>
-              {language}
-            </ListItemText>
-          </ListItem>
-        ))}
-      </List>
+      <React.Fragment>
+        <List disablePadding>
+          {['Swift', ...Object.keys(BUGZILLA_LANGUAGES)].map(language => (
+            <ListItem
+              className={classNames({
+                [classes.active]: language.toLowerCase() === activeLanguage,
+              })}
+              button
+              onClick={this.props.onLanguageClick}
+              id={language}
+              key={language}
+              component={Link}
+              to={`/languages/${language.toLowerCase()}`}>
+              <ListItemIcon>{icons[language] || <WebIcon />}</ListItemIcon>
+              <ListItemText disableTypography className={classes.text}>
+                {language}
+              </ListItemText>
+            </ListItem>
+          ))}
+        </List>
+        <div className={classes.link}>
+          <a className={classes.a} href="https://summerofcode.withgoogle.com">
```

Comment:
@helfi92 should we put this in the `constants.js`?

Reply:
That's a really good idea :)